crates/tests/src/snapshots/fe_compiler_tests__features__aug_assign@add.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: crates/tests/src/features.rs
+assertion_line: 1779
```

Comment:
Not exactly sure why `assertion_line` wasn't part of the file before but it came up as I regenerated the files.